### DISCUSSION TITLE
Place timestamp before title in generated filenames for sorting in Explorer view

### DISCUSSION THREAD

**Discussion by ngokevin** [logancyang/obsidian-copilot#1168] [Q&A]
**Is your feature request related to a problem? Please describe.**

The timestamps are after the generated title. Causing the files in the output folder to not have any order.

![image](https://github.com/user-attachments/assets/21698d5e-5e2d-4667-8715-bdc1e0816629)

**Describe the solution you'd like**

Filenames should start with the timestamp.

**Describe alternatives you've considered**

Manual renaming of files.

**Additional context**

**Comment by logancyang** [comment]
The conversations are sorted by time in desc in "Load conversation" command, which is its intended usage. I don't plan to change the format of the filename anytime soon because of backward compatibility.
